src/web-components/KSLContainerElement.ts

```diff
@@ -3,6 +3,7 @@ import { KSLAddButtonElement } from './KSLAddButtonElement';
 import { KSLHighlightElement } from './KSLHighlightElement';
 import { createTemplateForCustomElement, getRenderingRootMetadata, getTotalScrollOffset } from '../utils/node';
 import { IPositionable } from './abstract/KSLPositionedElement';
+import { BaseZIndex } from './constants/z-index';
```

Comment:
Nitpick: all other files in the repo use camelCase or PascalCase (classes) naming convention. We should probably rename this file to `zIndex.ts` just to be consistent.